keyboards/kradoindustries/krado66/keymaps/vial/vial.json

```
@@ -0,0 +1,291 @@
+{
+  "name": "Krado66",
+  "vendorId": "0x4B72",
+  "productId": "0x6664",
+  "lighting": "qmk_rgblight",
+  "menus": [
+    "via/keymap",
+    "via/layouts",
+    "via/macros",
+    "via/save_load",
+    "core/qmk_rgblight"
+  ],
```

Attributes not used by Vial removed.
```suggestion
  "lighting": "qmk_rgblight",
```